embc-app/ViewModels/PrintReferral.cs

```diff
@@ -91,12 +91,13 @@ private string MapPrintEvacueeTypeCode(Models.Db.Evacuee evacuee)
 
     public class PrintReferral : Referral
     {
+        private readonly TimeZoneInfo pst = TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time");
```

Comment:
Will this handle PDT and PST?

Reply:
Good point, I know how to ensure PDT, let me add it

Reply:
Looks like the time PST time zone supports Daylight savings into the conversion.  So nothing has to be done for now.  If we move to a parameterized timezone, we'll have to consider it based on the timezone selected.